Source/WebKitLegacy/WebCoreSupport/WebResourceLoadScheduler.h

```diff
@@ -81,7 +82,7 @@ class WebResourceLoadScheduler final : public WebCore::LoaderStrategy {
     bool isSuspendingPendingRequests() const { return !!m_suspendPendingRequestsCount; }
     void isResourceLoadFinished(WebCore::CachedResource&, CompletionHandler<void(bool)>&&) final;
 
-    class HostInformation {
+    class HostInformation : public CanMakeWeakPtr<HostInformation> {
```

Comment:
I think we should also make this inherit from CanMakeCheckedPtr and store it CheckedPtr/CheckedRef whenever we're calling functions with it.

Reply:
Ok, fixed.